app/models/user.rb

```diff
@@ -2,7 +2,13 @@ class User < ApplicationRecord
 
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+
+  has_many :reviews
+  has_many :likes
+  has_many :reviews, through: :likes
+
```

Comment:
Line 8 is overriding line 6. 

If you need to grab all of the reviews that a user has liked, then you will need to define a custom association e.g. `has_many :liked_reviews, through: :likes, class_name: "Review"`

But if I havent seen any place where you call `current_user.reviews`, so you might be able to get rid of line 8 altogether.

Reply:
Removed this line and all tests pass; also could not find any `current_user.reviews`